eo-parser/src/main/antlr4/org/eolang/parser/Program.g4

```diff
@@ -288,3 +290,9 @@ HEX: '0x' [0-9a-f]+;
 
 NAME: [a-z][\p{Letter}\p{General_Category=Decimal_Number}_-]*;
 
+TEXT_BLOCK_MARK: '"""';
+TEXT_BLOCK:
+        TEXT_BLOCK_MARK
+        .*?
```

Comment:
@andreoss what happened with indentation here? We use 4 spaces everywhere in this file, not 8.

Reply:
@yegor256 It looks at the indentation of the first `"""` and trims spaces accordingly from each line

Reply:
@andreoss I meant the formatting you use in `.g4` file :) Look at how the file is formatted and don't break the style.

Reply:
@yegor256 Fixed.